tf_files/shared/modules/compose_configs/creds.tpl

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Don't put this here.  These are files for terraform.  Put everything compose related under `Docker/compose-files`